README.md

```diff
@@ -30,10 +38,10 @@ The initial choices of tools are:
 
 ## Sources
 
-The slack bot was based on:
+The slack bot was initially based on:
 > A simple Python-powered starter Slack bot
 > [the tutorial](https://www.fullstackpython.com/blog/build-first-slack-bot-python.html)
 
 ## Licence
 
-See [LICENCE](https://github.com/iwi/linkatos/blob/master/LICENSE)
+See [LICENCE](https://github.com/iwi/linkatos/blob/master/LICENCE)
```

Comment:
Now that I see it again, I'd say put a summary of what license is it so there is no need to open the file. Something like:

```
Linkatos is licenced under the MIT License. See [LICENCE](https://github.com/iwi/linkatos/blob/master/LICENCE)
```